README.md

```diff
@@ -1,19 +1,127 @@
 Intl.NumberFormat Unified API Proposal
 ======================================
 
-## Motivation
+## Background / Motivation
 
-There are many requests for adding number-formatting-related features to ECMA 402.  A few of them include:
+There are many requests for adding number-formatting-related features to ECMA 402. These include:
 
-- tc39/ecma402#200
-- tc39/ecma402#186
-- tc39/ecma402#164
-- tc39/ecma402#163
-- tc39/ecma402#95
-- tc39/ecma402#91
-- tc39/ecma402#37
-- tc39/ecma402#32
+- [Expose narrow currency symbol](https://github.com/tc39/ecma402/issues/200) -- Section I
+- [Add currency accounting format](https://github.com/tc39/ecma402/issues/186) -- Section IV
+- [Add scientific notation](https://github.com/tc39/ecma402/issues/164) -- Section III
+- [Add option to force sign](https://github.com/tc39/ecma402/issues/163) -- Section IV
+- [Add compact decimal notation](https://github.com/tc39/ecma402/issues/37) -- Section III
+- [Add measure unit formatting](https://github.com/tc39/ecma402/issues/32) -- Section II
 
-Rather than complicate `Intl` with more subclasses with heavilly overlapping functionality, this proposal is to restructure the spec of `Intl.NumberFormat` to make it more easilly support additional features in a "unified" way.
+These features are important to both end users and to Google.  Since most of these features require carrying along large amounts of locale data for proper i18n support, exposing these features via a JavaScript API reduces bandwidth and lowers the barrier to entry for i18n best practices.
+
+Rather than complicate `Intl` with more constructors with heavilly overlapping functionality, this proposal is to restructure the spec of `Intl.NumberFormat` to make it more easilly support additional features in a "unified" way.
+
+Additional background: [Unified API for number formatting](https://github.com/tc39/ecma402/issues/215)
+
+## I. Spec Cleanup
```

Comment:
I like how the later sections include code samples demonstrating their use. Do you think you could add a code sample here using `currencyDisplay: "narrowSymbol"`?

I don't think it's necessary to include the unobservable spec refactorings in this document. If you want to be clear about the changes, I'd recommend making this a PR of 5 patches, each of which has a detailed commit message including things about the editorial/refactoring changes. Right now, it doesn't really look like the patches correspond to these 5 bullet points.

Reply:
Sorry, I just meant, "if". In my opinion, the most important thing is the observable changes, and you can just omit the description of the spec refactoring. (By bullets, I just meant the `## I.` headings.)